### ISSUE TITLE
django dataload > 'NoneType' object has no attribute 'get_parent_pk'

### ISSUE THREAD

**Issue by damienSD** [fabiocaccamo/django-treenode#88]
**Python version**
3.11

**Django version**
4.1.7

**Package version**
0.19.0

**Current behavior (bug description)**

With Simple Model with no relation
```python
class MyModel(models.Model, TreeNodeModel):
    ...
    name: str = models.CharField(max_length=50)
    treenode_display_field = "name"
    ...
```
After some inserts with parentals relations, then a django dumpdata with this model only : 
```bash
python manage.py dumpdata MyModel --output=my_model.json
```

on a standart django dataload : 
```bash
python manage.py loaddata my_model
```

```bash
> AttributeError: Problem installing fixture 'my_model.json': 'NoneType' object has no attribute 'get_parent_pk'
```

**Expected behavior**
> dataload OK

<a href="https://dashboard.polar.sh/fabiocaccamo/django-treenode/issues/88">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://dashboard.polar.sh/api/github/fabiocaccamo/django-treenode/issues/88/pledge.svg?darkmode=1">
  <img alt="Fund with Polar" src="https://dashboard.polar.sh/api/github/fabiocaccamo/django-treenode/issues/88/pledge.svg">
</picture>
</a>

**Comment by fabiocaccamo** [comment]
@damienSD thank you for reporting this, could you attach the `my_model.json` fixture you're trying to load please?

**Comment by fabiocaccamo** [comment]
@damienSD to speed up the debug of this issue, could you:
- attach the `my_model.json` fixture you're trying to load please?
- submit a PR with a failing test case?

Any of the above would be really appreciated!

**Comment by fabiocaccamo** [comment]
@damienSD this has been fixed in `0.20.1` version.
